serverless_template.yml

```diff
@@ -48,16 +52,18 @@ provider:
       Action:
         - "lambda:InvokeFunction"
       Resource:
-        - "arn:aws:lambda:::${self:custom.imageUploaderBucket}/*"
+        - "arn:aws:lambda:::${self:custom.imageUploadBucket}/*"
+
 custom:
   logRetentionInDays: 14
   defaultStage: dev
   customDomain:
-    domainName: image-service-${self:provider.stage}.pinster.io
-    basePath: ''
+    domainName: images.image-service-${self:provider.stage}.pinster.io
+    basePath: '(none)'
     stage: ${self:provider.stage}
     createRoute53Record: true
-    certificateName: image-service-dev.pinster.io
+    certificateName: images.image-service-dev.pinster.io # This is the name of the cert, it covers prod too.
```

Comment:
I changed the URLs for how we upload and generate thumbnails to `images.` The reason for this is that I wanted to use `image-service-<stage>` for our actual image hosting, and that meant it was way more of a pain to make that subdomain do everything. So I just made a new one. One caveat is that I changed the paths a bit for those too, I'll comment in those locations.

Reply:
👍